db/dexie_implementation.go

```diff
@@ -241,6 +241,22 @@ func (db *DB) AddMiniHeaders(miniHeaders []*types.MiniHeader) (added []*types.Mi
 	return dexietypes.MiniHeadersToCommonType(jsAdded), dexietypes.MiniHeadersToCommonType(jsRemoved), nil
 }
 
+// ResetMiniHeaders deletes all of the existing miniheaders and then stores new
+// miniheaders in the database.
+func (db *DB) ResetMiniHeaders(miniHeaders []*types.MiniHeader) (err error) {
```

Comment:
nit: I think we should either rename the parameter or change the comment to clarify that the "new miniheaders" means the given `miniHeaders` argument. I'm leaning toward the former.

```suggestion
func (db *DB) ResetMiniHeaders(newMiniHeaders []*types.MiniHeader) (err error) {
```